### DISCUSSION TITLE
Make snapshot list sortable

### DISCUSSION THREAD

**Discussion by flexwende** [grafana/grafana#58112] [General]
For better handling of the snapshots the list should be sortable. This could be made possible similar to the sorting options in "Browser" or "Library Panels" with the "Sort"-dropdown.

![grafik](https://user-images.githubusercontent.com/114215545/199678264-6b64eac5-24d6-4240-9116-3732be51c976.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
